web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -78,25 +82,31 @@ const FeedbackRequestPage = () => {
   const classes = useStyles();
   const location = useLocation();
   const history = useHistory();
-
   const query = queryString.parse(location?.search);
 
   const stepQuery = query.step?.toString();
+  const templateQuery = query.template?.toString();
+  const fromQuery = query.from?.toString();
+  const dueQuery = query.due?.toString();
+
+  const getStep = useCallback(() => {
+    if(!stepQuery || stepQuery < 1 || !/^\d+$/.test(stepQuery))
+      return 1;
+    else return parseInt(stepQuery);
+  },[stepQuery]);
 
-  let sendDate = query?.sendDate ? query.sendDate: todayDate.toString();
-  let dueDate = query?.dueDate ? query.dueDate: null
-  let activeStep = location?.search ? parseInt(stepQuery) : 1;
-  const numbersOnly = /^\d+$/.test(stepQuery);
```

Comment:
Deleting `sendDate` and `dueDate` here prevents the front-end from building, as they are referenced below in `<SelectDate>`